.travis.yml

```diff
@@ -55,17 +55,17 @@ before_install:
   - 'if [[ $ssl == "" ]]; then export ssl=off; fi'
   - 'if [[ $crypto == "" ]]; then export crypto=built-in; fi'
   - 'if [[ $TRAVIS_OS_NAME == "osx" && ( "$tests" == "1" || "$sim" == 1 || "$examples" == "1" || "$tools" == "1" || "$python" == "1" ) ]]; then
-    travis_retry brew update > /dev/null && brew install --quiet ccache boost-build;
+    travis_retry brew update > /dev/null && brew install ccache boost-build;
     fi'
-  - 'if [[ $TRAVIS_OS_NAME == "osx" && "$python" == "1" ]]; then travis_retry brew update > /dev/null && brew install --quiet boost-python; fi'
+  - 'if [[ $TRAVIS_OS_NAME == "osx" && "$python" == "1" ]]; then travis_retry brew install boost-python; fi'
   - 'if [ "$docs" = "1" ]; then
-    travis_retry brew install --quiet docutils;
+    travis_retry brew install docutils;
     mkdir -p /Users/travis/Library/Python/2.7/lib/python/site-packages;
     echo ''import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")'' >> /Users/travis/Library/Python/2.7/lib/python/site-packages/homebrew.pth;
     sudo easy_install Pygments;
     sudo easy_install -U aafigure;
-    travis_retry brew install --quiet graphviz;
-    travis_retry brew install --quiet Homebrew/python/pillow;
+    travis_retry brew install graphviz;
+    travis_retry brew install Homebrew/python/pillow;
```

Comment:
I wonder if merging these two calls into a singe ``brew install`` would make it more likely to pass too